### ISSUE TITLE
Copied 

### ISSUE THREAD

**Issue by allfred1** [iamgio/chorus#48]
Hello! I recently encountered a bug on windows 11 that I can't copy more than once. i.e. I copied a piece of code once, but I can't the second time I can only paste.
> windows 11 last on 15 December, the last version of the program

**Comment by allfred1** [comment]
**p.s** Sometimes I can't even copy and paste the code fragment that is in the editor

**Comment by iamgio** [comment]
@allfred1 I've been trying to reproduce the issue for the last couple of days with no luck (Windows 10 though). Also it isn't Chorus itself that handles copy-pasting, [RichTextFX](https://github.com/FXMisc/RichTextFX) is. Have you tried copy-pasting using the "Edit" menu bar instead of keyboard shortcuts?

**Comment by allfred1** [comment]
> @allfred1 I've been trying to reproduce the issue for the last couple of days with no luck (Windows 10 though). Also it isn't Chorus itself that handles copy-pasting, [RichTextFX](https://github.com/FXMisc/RichTextFX) is. Have you tried copy-pasting using the "Edit" menu bar instead of keyboard shortcuts?
> 
> That said, I am looking forward to updating the library, in case it was already fixed.

Yes, I tried, but it doesn't work, it's grayed out, it doesn't click and there's no reaction.

**Comment by allfred1** [comment]
> @allfred1 I've been trying to reproduce the issue for the last couple of days with no luck (Windows 10 though). Also it isn't Chorus itself that handles copy-pasting, [RichTextFX](https://github.com/FXMisc/RichTextFX) is. Have you tried copy-pasting using the "Edit" menu bar instead of keyboard shortcuts?
> 
> That said, I am looking forward to updating the library, in case it was already fixed.

And when I use the code search it works fine everything is inserted

**Comment by allfred1** [comment]
When will there be an update and will there be one at all?

**Comment by iamgio** [comment]
@allfred1 I'm still unable to reproduce the issue. I'm investigating.
